### ISSUE TITLE
Show current enabled pipeline in F3`

### ISSUE THREAD

**Issue by heymanMC** [grondag/canvas#299]
It would be helpful for debug purposes if Canvas shows the current selected pipeline in the F3 screen, similar to the way Optifine or Iris shows the current shaderpack in F3. A Canvas version number would be helpful as well.

**Optifine's Implementation:**
![image](https://user-images.githubusercontent.com/38384673/124861647-377a6a00-df71-11eb-955d-fd0bb53c4929.png)

**Iris' Implementation:**
![image](https://user-images.githubusercontent.com/38384673/124861858-993ad400-df71-11eb-9b08-7c89004cccfc.png)

**Comment by grondag** [comment]
Yes, I'd like to see to see these added. I'd also like an option to display FPS and some other metrics in some screen corner without F3, but I guess that's a different feature.
